src/regions.ts

```diff
@@ -1 +1 @@
-export const standardRegions = ['us-east-1', 'us-east-2', 'us-west-2', 'eu-west-1', 'eu-central-1'];
+export const standardRegions = ['us-east-1', 'us-east-2', 'us-west-2', 'eu-west-1', 'eu-central-1', 'ap-southeast-2', 'ap-northeast-1'];
```

Comment:
```suggestion
export const standardRegions = [
  'us-east-1',
  'us-east-2',
  'us-west-2',
  'eu-west-1',
  'eu-central-1',
  'ap-southeast-2',
  'ap-northeast-1',
];
```

This is to fix the lint error.